realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -209,7 +210,7 @@ public static void setDefaultSessionErrorHandler(@Nullable SyncSession.ErrorHand
      * @return the {@link SyncSession} for the specified Realm.
      * @throws IllegalArgumentException if syncConfiguration is {@code null}.
      */
-    public static synchronized SyncSession getSession(SyncConfiguration syncConfiguration) {
+    public static synchronized SyncSession getSession(SyncConfiguration syncConfiguration, URI... resolvedRealmURL) {
```

Comment:
This API is public. I don't assume that was the intent? 

I can see the issue though, the `SyncManager` is in `io.realm` while the facade is in `io.realm.internal`. I'm trying to remember why the facade isn't just package private in `io.realm`? I think I tried at some point, but something else failed. In any case, there is a lot of TODO/FIXME's in the Facade about it.

Reply:
it's a variadic function, so shouldn't break the API

Reply:
It doesn't break it, but it is totally non-obvious what this paramter do. I would still pull this signature out in its own method, and then document it as internal so we can remove it later if at all possible?